docker-compose_developer.yml

```diff
@@ -46,6 +58,9 @@ services:
 
   elasticsearch:
     image: elasticsearch:7.14.1
+    profiles:
```

Comment:
don't we need elasticsearch in every case? even for backend development we would need to have infrastructure for our backend-components (e.g. analyzer) that are running within a local spring-boot app

Reply:
It depends on what exactly is being developed. The profiles I suggested worked in my use case, but might not be suitable for other cases. Also, my use of "backend" and "frontend" seems very misleading. Perhaps we should discuss which profiles are generally useful? My understanding is that the following components could be developed outside of docker-compose:

- camunda-bpm-platform
- polling client
- event pipeline
- analyzer
- web-app
 
So we could introduce profiles that each exclude the respective component and start up the remaining ones?